src/util/sandbox.h

```diff
@@ -53,17 +53,27 @@ class Sandbox {
         return createSecurityToken(dir.canonicalPath(), true);
     }
 
-    static SecurityTokenPointer openSecurityToken(mixxx::FileInfo* pFileInfo, bool create);
-    static SecurityTokenPointer openSecurityTokenForDir(const QDir& dir, bool create);
+#if defined(__GNUC__) && __cplusplus < 201907L
+#define RATIONALE
```

Comment:
Introducing a define with a generic name in a widely used header file is probably not a good idea.

Reply:
I agree, I forgot this was the header and not the implementation file when I saw this. I've renamed it to `SECURITY_TOKEN_NODISCARD_RATIONALE`.